setup.sh

```diff
@@ -94,9 +123,16 @@ cd ${DIR}/${NAME}/src
 
 # cmsenv
 eval `scramv1 runtime -sh`
-git cms-init $ACCESS_CMSSW
+git cms-init $ACCESS_CMSSW $BATCH
 git config gc.auto 0
 
+# batch git config
+if [[ ! -z "$BATCH" ]]; then
```

Comment:
`-n` is a little clearer than `! -z` (to me, anyway)

Reply:
Switched